docs/source/getting-started.mdx

```diff
@@ -40,39 +35,48 @@ Your project directory now contains a `package.json` file.
 
 ## Step 2: Install dependencies
 Applications that run Apollo Server require two top-level dependencies:
-<!-- TODO: CHECK THIS LINK once packaged published -->
 * [`@apollo/server`](https://www.npmjs.com/package/@apollo/server) is the core library for Apollo Server itself, which helps you define the shape of your data and how to fetch it.
 * [`graphql`](https://npm.im/graphql) is the library used to build a GraphQL schema and execute queries against it.
 
-Run the following command to install both of these dependencies and save them in
+We'll also install the [`graphql-tag`](https://www.npmjs.com/package/graphql-tag) package, which includes a template literal tag that parses GraphQL query strings into the standard GraphQL AST.
+
+Run the following command to install all of these packages and save them in
 your project's `node_modules` directory:
 
 ```bash
-npm install @apollo/server graphql
+  npm install @apollo/server graphql graphql-tag
 ```
 
-Also create an empty `index.ts` file in your project's root directory:
+If you are using JavaScript, create a `src/index.js` file. If you plan to use TypeScript, create a `src/index.ts` file:
 
 ```bash
-touch src/index.ts
+  touch src/index.ts
 ```
 
-To keep things simple, `index.ts` will contain **all** of the code for this example application.
+To keep things simple, this `index` file will contain **all** of the code for our example application.
+
+<ExpansionPanel title="Learn how to set up TypeScript (Optional)">
 
-### Set up TypeScript
-Next we'll set up and configure TypeScript to compile our code. We'll install `typescript`:
+#### Set up TypeScript
+
+1. If you do not already have TypeScript installed on your computer, install it globally now:
+```bash
+npm install -g typescript
+```
+
+2. Run the following command to install TypeScript into your project's dependencies:
 
 ```bash
 npm install typescript
 ```
```

Comment:
We've mostly moved away from global installs. In the more recent years, `npx` has made this a lot nicer.

My suggestion would be this:
Step 1: Install `typescript` as a _dev_ dependency via `npm install --save-dev typescript` (or `npm i -D typescript`) for short
Step 2: Run `npx tsc --init`

If we recommend running `tsc ...` anywhere further down, it would instead be `npx tsc ...`

Reply:
I'll also note that `npx tsc --init` gives us a massive tsconfig file with a bunch of unneeded things commented out. We might be better off giving them a simple config to copy/paste.